src/plugin-host/plugin-events/plugin-event-arguments.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Plugin Event Messages are used to communicate data
+ * between a Plugin Host and its associated plugins.
+ */
+export abstract class PluginEventArguments
```

Comment:
Considering this is an empty abstract class, can't it also be just an interface instead, or are we restricted here by the language?

Reply:
Sure it could.